rfcs/026-work-enricher/README.md

```diff
@@ -0,0 +1,160 @@
+# RFC 026: Work Enricher
+
+## Background
+
+Recently we have started ingesting archive data from Calm. An archive according to our model is a collection of works arranged hierarchically in a tree structure according to some path. For example, the works with paths `A`, `A/1`, `A/1/X`, `A/1/Y` and `A/2` would for the following tree:
+
+```
+A
+├── 1
+│   ├── X
+│   └── Y
+│
+└── 2
+```
+
+Our API allows us to display relations for some particular work using a number of fields: 
+
+* `parts` is an array containing the children of the work (i.e. `parts` for work `A` would contain `A/1` and `A/2`)
+
+* `partOf` is an array containing the ancestors of the work (i.e. `partOf` for work `A/1/X` would contain `A` and `A/1`)
+
+* `precededBy` is an array containing the siblings of the work earlier in sort order (i.e. `precededBy` for work `A/2` would contain `A/1`)
+
+* `succeededBy` is an array containing the siblings of the work later in sort order (i.e. `succeededBy` for work `A/1/X` would contain `A/1/Y`)
```

Comment:
Two notes:

* You probably want "e.g." (for example) instead of "i.e." (that is)
* Double-checking I've understood correctly, and maybe expanding the example would benefit: `precededBy` doesn't show the siblings that come later, right? And vice versa for `succeededBy`?

So if the tree was

```
A
 ├── 1
 │   ├── X
 │   ├── Y
 │   └── Z
 │
 └── 2
```

Then the `precededBy` of A/1/Y would be {A/1/X}, and the `succeededBy` would be {A/1/Z}?

Reply:
Yep exactly :+1: